<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserLogin</class>
 <widget class="QWidget" name="UserLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Image/webp.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>25</y>
     <width>320</width>
     <height>235</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
font: 18pt &quot;文泉驿等宽微米黑&quot;;
border: 2px solid rgb(14, 237, 237);
border-radius:8px;
margin-top:10px;
}
QGroupBox:title{
color:white;
subcontrol-origin: margin;
subcontrol-position: top center;
left: 5px;
}
QLabel{
color: rgb(255, 255, 255);
}
QPushButton#pBtn_ok{
font: 16pt &quot;文泉驿等宽微米黑&quot;;
border:1px solid rgb(85, 0, 127);
border-radius:10px;
color: rgb(0, 0, 0);
background-color: rgb(85, 255, 127);
}
QPushButton#pBtn_ok:pressed{
font: 14pt &quot;文泉驿等宽微米黑&quot;;
}

QPushButton#pBtn_cancel{
font: 16pt &quot;文泉驿等宽微米黑&quot;;
border:1px solid rgb(85, 0, 127);
background-color: rgb(167, 0, 0);
border-radius:10px;
color: rgb(255, 255, 255);
}
QPushButton#pBtn_cancel:pressed{
font: 14pt &quot;文泉驿等宽微米黑&quot;;
}

</string>
   </property>
   <property name="title">
    <string>用户登录</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QPushButton" name="pBtn_cancel">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>195</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>取  消</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_passwd">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>150</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>输入密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn_ok">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>195</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>确  定</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_msg">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>50</y>
      <width>241</width>
      <height>56</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>欢迎您登录本系统
请输入该设备的密码...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_cue">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>150</y>
      <width>66</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>密 码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>25</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;文泉驿等宽微米黑&quot;;</string>
    </property>
    <property name="text">
     <string>产品编号:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_hostSerial">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>25</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;文泉驿等宽微米黑&quot;;
border:1px;

</string>
    </property>
    <property name="text">
     <string>F1-11-201702-003</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_cue_2">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>110</y>
      <width>66</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>用 户：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>110</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	border:1px solid  rgb(53, 156, 212);
	color: rgb(255, 255, 255);
	font: 14pt &quot;文泉驿等宽微米黑&quot;;
   background-color: rgb(0, 50, 105);
}
QComboBox::item {	
	font: 14pt &quot;文泉驿等宽微米黑&quot;;
	color: rgb(255, 255, 255);
    background-color: rgb(0, 50, 105);
	selection-background-color: rgb(0, 170, 255);
}</string>
    </property>
    <item>
     <property name="text">
      <string>操作员</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>管理员</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>注 销</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_key">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>150</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>输入解锁密钥</string>
    </property>
   </widget>
   <zorder>lineEdit_key</zorder>
   <zorder>pBtn_cancel</zorder>
   <zorder>lineEdit_passwd</zorder>
   <zorder>pBtn_ok</zorder>
   <zorder>lb_msg</zorder>
   <zorder>lb_cue</zorder>
   <zorder>label_10</zorder>
   <zorder>lb_hostSerial</zorder>
   <zorder>lb_cue_2</zorder>
   <zorder>comboBox</zorder>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>450</y>
     <width>340</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>425</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽微米黑</family>
     <pointsize>22</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
